data/seedData.py

```diff
@@ -68,45 +68,48 @@ def seedData():
         archived=False,
     )
     user_gray_pouponn.save_to_db()
-
-    user_anthony_redding = UserModel(
-        email="pending1@dwellingly.org",
-        role=RoleEnum.PENDING,
-        firstName="Anthony",
-        lastName="Redding",
-        password="1234",
-        phone="555-555-5555",
-        archived=False,
+    user_anthony_redding = UserModel.create(
+        payload={
+            "email": "pending1@dwellingly.org",
+            "firstName": "Anthony",
+            "lastName": "Redding",
+            "password": "1234",
+            "phone": "555-555-5555",
+        },
+        schema=UserRegisterSchema,
     )
     user_anthony_redding.save_to_db()
```

Comment:
For each of these four users that you changed over to use the schema, we don't need the `save_to_db()` line anymore, since `UserModel.create()` already saves it to the db.  Let's remove this line.

Once that line is removed, flake8 will complain that you have unused variables, so just remove the variable assignments:
```python
UserModel.create(
    payload={
        "email": "pending1@dwellingly.org",
        "firstName": "Anthony",
        "lastName": "Redding",
        "password": "1234",
        "phone": "555-555-5555",
    },
    schema=UserRegisterSchema,
)
```